pipelines.go

```diff
@@ -406,3 +406,35 @@ func (s *PipelinesService) DeletePipeline(pid interface{}, pipeline int, options
 
 	return s.client.Do(req, nil)
 }
+
+// UpdateMetadataOptions represents the available UpdateMetadata() options.
+//
+// https://docs.gitlab.com/ee/api/pipelines.html#update-pipeline-metadata
+type UpdateMetadataOptions struct {
+	Name string `url:"ref" json:"name"`
+}
+
+// UpdateMetadata You can update the metadata of a pipeline. The metadata contains the name of the pipeline.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/pipelines.html#update-pipeline-metadata
+func (s *PipelinesService) UpdateMetadata(pid interface{}, pipeline int, opt *UpdateMetadataOptions, options ...RequestOptionFunc) (*Pipeline, *Response, error) {
```

Comment:
If we follow the conventions in the docs, this should be:
```suggestion
func (s *PipelinesService) UpdatePipelineMetadata(pid interface{}, pipeline int, opt *UpdatePipelineMetadataOptions, options ...RequestOptionFunc) (*Pipeline, *Response, error) {
```